soh/soh/z_message_OTR.cpp

```diff
@@ -144,4 +144,13 @@ extern "C" void OTRMessage_Init()
             "Vous obtenez un %rQuart de&Coeur%w! Vous en avez collecté&%r{{heartPieceCount}}%w en tout!"
         }
     );
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_MARKET_GUARD_NIGHT,
+        {
+            TEXTBOX_TYPE_BLACK, TEXTBOX_POS_BOTTOM,
+            "You look bored. Wanna go out for a&walk?\x1B&%gYes&No%w",
+            "Du siehst gelangweilt aus.&Willst du einen Spaziergang machen?\x1B&%gJa&Nein%w",
+            "Tu as l'air de t'ennuyer. Tu veux&aller faire une promenade?\x1B&%gOui&Non%w",
```

Comment:
> Replace DeepL translations with actual ones

Probably doesn't need to block this PR but add comments here indicating this is AI translated